src/DssSpell.t.sol

```diff
@@ -194,19 +194,45 @@ contract DssSpellTest is DssSpellTestBase {
         );
     }
 
-    function testIlkClipper() private { // make private to disable
+    function testIlkClipper() public { // make private to disable
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // XXX
         _checkIlkClipper(
-            "XXX-A",
-            GemJoinAbstract(addr.addr("MCD_JOIN_XXX_A")),
-            ClipAbstract(addr.addr("MCD_CLIP_XXX_A")),
-            addr.addr("MCD_CLIP_CALC_XXX_A"),
-            OsmAbstract(addr.addr("PIP_XXX")),
-            5_000 * WAD
+            "LINK-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_LINK_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_LINK_A")),
+            addr.addr("MCD_CLIP_CALC_LINK_A"),
+            OsmAbstract(addr.addr("PIP_LINK")),
+            1_000_000 * WAD
+        );
+
+        _checkIlkClipper(
+            "MATIC-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_MATIC_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_MATIC_A")),
+            addr.addr("MCD_CLIP_CALC_MATIC_A"),
+            OsmAbstract(addr.addr("PIP_MATIC")),
+            10_000_000 * WAD
+        );
+
+        _checkIlkClipper(
+            "YFI-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_YFI_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_YFI_A")),
+            addr.addr("MCD_CLIP_CALC_YFI_A"),
+            OsmAbstract(addr.addr("PIP_YFI")),
+            1_000 * WAD
+        );
+
+        _checkIlkClipper(
+            "UNIV2USDCETH-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_UNIV2USDCETH_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_UNIV2USDCETH_A")),
+            addr.addr("MCD_CLIP_CALC_UNIV2USDCETH_A"),
+            OsmAbstract(addr.addr("PIP_UNIV2USDCETH")),
+            1 * WAD
```

Comment:
Are these amounts just arbitrary?

Reply:
Yes. They just have to be higher than the `dust` for the collateral and lower than global debt ceiling. The number is then used to create a new vault to liquidate it and take created auction. The test is written in way that they are not computed inside the test, but have to be hardcoded